### DISCUSSION TITLE
[SC] Atti di Nascita: Nuovi campi per AIRE e flusso Notifiche

### DISCUSSION THREAD

**Discussion by AMonopoliADS** [italia/ansc#978] [Supporto servizi cooperativi]
salve.

chiedo un chiarimento / conferma sull'uso del metaDato flagTrasmissioneResidenzaEstera all'interno delle sezioni "padre" e "madre", nell'ambito degli atti di nascita.

in occasione dell'ultima call è stato chiarito che il metaDato in questione, utilizzato principalmente per gli automatismi della web-app, nell'ambito dei web-service può essere passato sempre a 'false'.
è stato chiesto anche - considerata la poca utilità nell'integrazione via web-service - se potesse essere evitato l'invio del campo all'interno del json e - a memoria - mi sembra che la risposta sia stata affermativa.

dalle prove effettuate, invece, riscontro che l'assenza del metaDato nel json compromette il flusso di notifica già verso ANSC (e conseguentemente anche quello verso ANPR)

potete verificare ad esempio l'atto con idAnsc 2024-3113632-216-039010 (comunicazione di validazione 2393660).
l'atto in questione è stato inviato con i dati AIRE sia per l'intestatario che per la madre, ma senza passare il flagTrasmissioneResidenzaEstera.
Per quest'atto non è arrivata la notifica ANSC di "trascrizione"
(ritengo che la condizionante sia quella descritta, perchè per atti successivi con il metaDato inviato - anche se a 'false' - le nofifiche sono arrivate)

potete chiarire se il comportamento descritto è quello che avete previsto, oppure se l'assenza del metaDato genera un bug nel sistema di notifica?

grazie
Alessandro ADS

**Comment by asqueglia** [comment]
Buongiorno, abbiamo verificato che il problema riscontrato dipende da un bug sul sistema che provvediamo quanto prima a sanare. Il comportamento confermiamo che è quello che abbiamo indicato e specificato in call.

**Comment by asqueglia** [comment]
Correttiva rilasciata nella versione 1.32.0 #979
